sys/mobileapps.yaml

```diff
@@ -51,7 +51,15 @@ paths:
               uri: /{domain}/sys/key_value/mobileapps.remaining/{title}
               headers: '{{from_service.headers}}'
               body: '{{from_service.body.remaining}}'
-        - return:
+        - purge_caches:
+            request:
+              method: post
+              uri: /{domain}/sys/events/
+              body:
+                - uri: //{{request.params.domain}}/api/rest_v1/page/mobile-sections/{request.params.title}
+                - uri: //{{request.params.domain}}/api/rest_v1/page/mobile-sections-lead/{request.params.title}
+                - uri: //{{request.params.domain}}/api/rest_v1/page/mobile-sections-remaining/{request.params.title}
```

Comment:
@gwicke we have `request` everywhere in the code, could replace all at once later for consistency